content/updates/2020-07/consistency-in-structure.md

```diff
@@ -0,0 +1,15 @@
+# Avoid spelling mistakes in your product data
```

Comment:
Add the YAML before the markdown for the PIM communication channel. See how to do it right here: https://www.notion.so/akeneo/Write-a-great-Serenity-updates-announcement-df78993e3fc746cda7d480e1a941048f